force_wfmanager/wfmanager.py

```diff
@@ -1,16 +1,27 @@
-from pyface.tasks.api import TaskWindowLayout
+import logging
+
 from envisage.ui.tasks.api import TasksApplication, TaskWindow
-from pyface.api import (ConfirmationDialog, YES, NO, CANCEL)
+
+from pyface.api import (CANCEL, ConfirmationDialog, NO, YES)
+from pyface.tasks.api import TaskWindowLayout
+
+log = logging.getLogger(__name__)
 
 
 class WfManager(TasksApplication):
     id = 'force_wfmanager.wfmanager'
     name = 'Workflow Manager'
 
+    def __init__(self, plugins):
```

Comment:
If you want to keep an __init__ method here, please use keyword arguments. Something like 

```
    def __init__(self, plugins=None):
```

If you remove your __init__, then the traits init will still work!!! Right now, it won't.

```
plugins = [MyPlugin]
manager = WfManager(plugins) # works with your init but not traits init
manager = WfManager(plugins=plugins) # with with Traits but not yours 
```